docs/manual-configuration.md

```diff
@@ -0,0 +1,37 @@
+# Manually configure the collector
+Collecting logs and host metrics with the Elastic Distribution for the OpenTelemetry Collector without using the [guided onboarding](docs/guided-onboarding.md) requires some manual configuration.
+
+## Before you begin
+The onboarding has the following requirements and limitations:
+
+- The **Admin** role or higher is required to onboard system logs and metrics. To learn more, refer to <DocLink slug="/serverless/general/assign-user-roles" />.
+- Root privileges on the host are required to run the OpenTelemetry collector used in this quickstart.
+- The collector only work on Kubernetes, Linux, and MacOS systems.
+- Refer to [Elastic OpenTelemetry Collector limitations](collector-limitations.md) for known limitations when using the Elastic Distribution for the OpenTelemetry collector.
+
+## Collect your logs and metrics
+
+To manually configure the collector, gather the following information:
+
+- **Your Elasticsearch endpoint**: From the help menu in Kibana or your serverless Observability project, select **Connection details** and copy the **Elasticsearch endpoint**.
+- **API key**:
+   - **Kibana:** From the help menu, select **Connection details** and select **Create and manage API keys**. From the **API keys** page, select **Create API key**. Give your API key a name, select **Create API key**, and copy the new API key.

```

Comment:
By the "help menu" do you mean the 🛟 icon? If yes, I don't see "Connection details" as an option.

Reply:
Not sure which icon you put because it's showing up like this:
![image](https://github.com/user-attachments/assets/fbce0ac4-4e51-47b2-83e1-d6aaf23b87ad)

But I mean this icon:
![image](https://github.com/user-attachments/assets/cabd56f5-acfe-435c-b85e-fd941380be6a)

Would probably work better if I put the actual icon.